.github/workflows/update-artifacts.yml

```diff
@@ -106,50 +106,79 @@ jobs:
       - name: Check if Truefoundry Chart Images Support Multiple Architectures
         run: |
           python scripts/generate-artifacts-manifest/check_chart_images_support_multiple_architectures.py \
-          charts/tfy-k8s-aws-eks-inframold/artifacts-manifest.json \
-          scripts/generate-artifacts-manifest/multi-arch-image-whitelist.json
+            charts/tfy-k8s-aws-eks-inframold/artifacts-manifest.json \
+            scripts/generate-artifacts-manifest/multi-arch-image-whitelist.json
+
 
       - name: Generate Changelog
```

Comment:
Please add a comment before this step that changelog is calculated only for one of the inframolds only